Tests/Fuzzers/FuzzTarget.cpp

```diff
@@ -54,18 +54,21 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
 		return 1;
 	}
 
-	// parse the raw packet into a parsed packet
-	pcpp::Packet parsedPacket(&rawPacket);
-
-	// verify the packet is IPv4
-	if (parsedPacket.isPacketOfType(pcpp::IPv4))
+	while (reader.getNextPacket(rawPacket))
```

Comment:
If we do the while loop, we lose the first packet which is read in line 51 (`reader.getNextPacket(rawPacket)`)...

Reply:
Good catch. Made do/while.